join.sh

```diff
@@ -70,3 +70,5 @@ docker run -t --rm \
         join \
             --hostname $(hostname) \
             --config-path /target$(realpath $1)
+
+touch /var/lib/prom.done
```

Comment:
For clarification, if the file `/var/lib/prom.done` exists, then DryDock won't run?   

This might be me thinking outside the box, and maybe something for later iterations, but would it make sense to make this a log file that lives in `/var/log/prom.log`   and maybe list some of the output we are writing to STDOUT currently when Promenade runs initially?